src/app/components/Navbar.jsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import { useState } from "react";
+
+import { Menu, X } from "lucide-react";
+import Image from "next/image";
+
+const Navbar = () => {
+  const [isClick, setisClick] = useState(false);
+  const toggleNavbar = () => {
+    setisClick(!isClick);
+  };
+  return (
+    <>
+      {/* eslint-disable-next-line @next/next/no-css-tags */}
+      <link href="./output.css" rel="stylesheet"></link>
+      <nav className=" fixed top-0 left-0 right-0 bg-black/10 backdrop-filter backdrop-saturate-180 backdrop-blur-sm rounded-3xl mx-9 my-5">
+        <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+          <div className="flex items-center justify-between h-16">
+            <div className="flex items-center">
+              <div className="flex-shrink-0">
+                <a href="/" className="text-white">
+                 <Image
+                  src="/images/Drama-Logo (Navbar).png"
+                    className="w-21 h-20 "
+                     alt="logo"
+                     width="160"
+                     height="20"
+                  />
+                </a>
+              </div>
+            </div>
+            <div className="hidden md:block">
+              <div className="ml-4 flex items-center space-x-4">
```

Comment:
create an array of objects for navlinks. map over the array to render the links. Also use Link component of Nextjs instead of anchor tag.